app/src/main/java/com/serwylo/lexica/game/Game.java

```diff
@@ -44,17 +44,31 @@
 
 public class Game implements Synchronizer.Counter {
 
+	public static final String SCORE_TYPE = "scoreType";
+	public static final String SCORE_WORDS = "W";
+	public static final String SCORE_LETTERS = "L";
+
 	private static final String TAG = "Game";
 	private int timeRemaining;
 	private int maxTime;
 
 	private int maxTimeRemaining;
 
 	private Board board;
+	private int score;
+	private String scoreType;
 
 	public enum GameStatus { GAME_STARTING, GAME_RUNNING, GAME_PAUSED, GAME_FINISHED }
 
-	public static final int WORD_POINTS[] = {
+	//TODO: i18n
+	private static final int[] LETTER_POINTS = {
+		//  A, B, C, D, E, F, G, H, I, J, K, L, M
+			1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3,
+		//  N, O, P, Qu,R, S, T, U, V, W, X, Y, Z
+			1, 1, 3, 5, 1, 1, 1, 1, 4, 4, 8, 4, 10
```

Comment:
Is this based on scrabble scoring? Looks like it mostly is, except for the Qu (which would be 11 - 10 for Q + 1 for u).

Reply:
It is Scrabble-based. I figured the Qu combo should be worth less since it always comes with a U. Doesn't really matter to me, honestly.

Reply:
That makes sense. I agree that during scrabble Q is worth more because it is only useful _if_ you have a U. Thanks for the clarification.